tests/TestCli.cpp

```diff
@@ -1021,7 +1021,7 @@ void TestCli::testInfo()
     QCOMPARE(m_stdout->readLine(), QByteArray("Number of short passwords: 0\n"));
     QCOMPARE(m_stdout->readLine(), QByteArray("Number of weak passwords: 2\n"));
     QCOMPARE(m_stdout->readLine(), QByteArray("Entries excluded from reports: 0\n"));
-    QCOMPARE(m_stdout->readLine(), QByteArray("Average password length: 11 characters\n"));
```

Comment:
you need to revert this change

Reply:
Sure, done ! May I ask what's up with the CI on this test ?

Reply:
Nothing it works fine, you got all green checks

Reply:
That would indicate your translation files on your computer are not working. The expected string is the English translated version of the base string, the one with `(s)`